src/breeze/components/breeze/flash_messages.cr

```diff
@@ -2,7 +2,7 @@ class Breeze::FlashMessages < BreezeComponent
   needs flash : Lucky::FlashStore
 
   def render
-    @flash.each do |flash_type, flash_message|
+    flash.each do |flash_type, flash_message|
```

Comment:
Decided to keep flash messages just in case some extension later on wants to use them... We can always remove it later if we decide it's not useful.